ragna/deploy/_api/schemas.py

```diff
@@ -26,14 +26,14 @@ def from_core(cls, document: ragna.core.Document) -> Document:
             name=document.name,
         )
 
-    def to_core(self) -> ragna.core.Document:
+    def to_core(self, metadata={}) -> ragna.core.Document:
```

Comment:
1. Do we even need a default value here?
2. If yes, don't set mutable objects as default values. 
    ```suggestion
        def to_core(self, metadata: Optional[dict[str, Any]] = None) -> ragna.core.Document:
        if metadata is None:
            metadata = {}
    ```